bedrock/firefox/templates/firefox/nightly/whatsnew-fx145.html

```diff
@@ -0,0 +1,49 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% extends "firefox/nightly/whatsnew.html" %}
+
+{% block page_css %}
+  {{ super() }}
+  {{ css_bundle('nightly_whatsnew_145') }}
+{% endblock %}
+
+{% set params = '?utm_source=firefox.com&utm_medium=referral&utm_campaign=wnp145nightly' %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+      <h1 class="wnp-content-title wnp-u-centered">Join the Mozilla community in the upcoming <strong>Ask a Fox</strong> event!</h1>
+
+      {{ resp_img(
+        url='img/firefox/whatsnew/ask-a-fox.png',
+        srcset={
+          'img/firefox/whatsnew/ask-a-fox-high-res.png': '2x'
+        },
+        optional_attributes={
+          'class': 'wnp-content-img',
+          'width': '250',
+          'height': '250',
+          'alt': 'Sticker illustration of a fox with a three-dot speech bubble above its head.'
+        }
+      ) }}
+
+      <p>The Mozilla Support platform (a.k.a. SUMO) receives over 20 million visits each month. It’s the go-to place for Firefox users to find troubleshooting tips, get help, and discover useful resources in multiple languages.</p>
+
+      <p>But it’s more than just a help site—it’s also a community. Here, you can connect with fellow Firefox fans from around the world. You might even come across contributors like <a href="https://blog.mozilla.org/sumo/2024/07/29/contributor-spotlight-jefferson-scher/{{ params }}">Jefferson</a> or <a href="https://blog.mozilla.org/sumo/2024/09/26/contributor-spotlight-noah-y/{{ params }}">Noah</a>, who have been sharing their expertise for more than a decade and have an incredible depth of knowledge about your favorite browser.</p>
```

Comment:
Normally the current style guide would call for spaces around the em dash, but given this copy is approved as-is I'd say we keep it that way.